src/bramble/client/main.js

```diff
@@ -204,6 +204,7 @@ define([
         self.getSidebarVisible = function() { return _state.sidebarVisible; };
         self.getRootDir = function() { return _root; };
         self.getWordWrap = function() { return _state.wordWrap; };
+        self.getAutoComplete = function() { return _state.allowAutoComplete; };
```

Comment:
Because you're changing the public API, please update the README as well, see https://github.com/mozilla/brackets#bramble-instance-getters